### PR TITLE
Update doc for verticle-deploy usage

### PR DESCRIPTION
**Pull request by EmadAlblueshi**
A tiny update for verticle-deploy usage

### CODE REVIEW

**Comment by tsegismont** [comment]
@EmadAlblueshi thanks for the PR. Vert.x docs are generated from the [package file](https://github.com/vert-x3/vertx-shell/blob/master/src/main/java/io/vertx/ext/shell/package-info.java). You should update this file and then regenerate the docs by building the project (include the generated files in your commit)

See the [How is managed vert.x documentation](https://github.com/vert-x3/wiki/wiki/How-is-managed-vert.x-documentation) wiki page for details.

**Comment by EmadAlblueshi** [comment]
@tsegismont Got it :+1:

**Comment by EmadAlblueshi** [comment]
@tsegismont is it ok ?

File: src/main/java/io/vertx/ext/shell/package-info.java [hunk 1]
```diff
@@ -291,7 +291,7 @@
  * . Verticle commands
  * .. verticle-ls: list all deployed verticles
  * .. verticle-undeploy: undeploy a verticle
- * .. verticle-deploy: deployes a verticle
+ * .. verticle-deploy: deployes a verticle with deployment options as JSON string
```
Comment (tsegismont) [inline src/main/java/io/vertx/ext/shell/package-info.java]: could you fix the typo ("deployes")?

Comment (EmadAlblueshi) [inline src/main/java/io/vertx/ext/shell/package-info.java]: Yes

Comment (EmadAlblueshi) [inline src/main/java/io/vertx/ext/shell/package-info.java]: @tsegismont Done

**Comment by tsegismont** [comment]
Thank you @EmadAlblueshi

**Comment by EmadAlblueshi** [comment]
@tsegismont You are more than welcome and I'm very glad to contribute :-)

**Pull request merged**